src/reducers/books.js

```diff
@@ -1,3 +1,21 @@
-const booksReducer = initialState => (state = initialState) => state;
+import actions from '../actions/index';
+
+const { action } = actions;
+const { CREATE_BOOK, REMOVE_BOOK } = action;
```

Comment:
### [Optional]
I don't think you really need these two lines.

And your `bookReduc` can simply be; 
`const bookReduc= (state = initalState, action) => {....}`